libs/commons-database/src/main/kotlin/gov/cdc/ocio/database/health/HealthCheckCouchbaseDb.kt

```diff
@@ -17,29 +17,23 @@ class HealthCheckCouchbaseDb: HealthCheckSystem("Couchbase DB"), KoinComponent {
 
     private val config by inject<CouchbaseConfiguration>()
 
+    // Lazily initialized Couchbase Cluster instance
+    private val cluster: Cluster by lazy {
+        Cluster.connect(config.connectionString, config.username, config.password)
```

Comment:
Did you try the health check when the database is unavailable?  Just wondering if Cluster.connect will throw an exception.  Doesn't look like it, but would be good to test if you haven't already.